lstchain/scripts/lstchain_find_pedestals.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python
+
+"""
+This script reads all DL1 subrun files (i.e. not merged) in the input
+directory, and finds the event_id's of interleaved pedestals. It writes them
+out into one hdf5 file for each sub-run, called pedestal_ids_RunXXXXX.YYYY.h5,
+which contains a single table with one column, 'event_id'
+"""
+
+import argparse
+import glob
+import os
+import numpy as np
+import pandas as pd
+
+from pathlib import Path
+from ctapipe.io import read_table
+from lstchain.paths import parse_dl1_filename
+
+parser = argparse.ArgumentParser(description="Interleaved Pedestal Finder")
+
+parser.add_argument('-f', '--input-dir', dest='srcdir',
+                    type=str, default='./',
+                    help='path to the directory of the DL1 files'
+                    )
+
+parser.add_argument('output_dir', metavar='output-dir', type=Path,
+                    help='Path where to store the output hdf5 files'
+                    )
+
+def main():
+
+    args = parser.parse_args()
+
+    files = glob.glob(args.srcdir+'dl1_LST-1.Run?????.????.h5')
+    if not files:
+        raise IOError("No input dl1 files found")
+    files.sort()
+
+    output_dir = args.output_dir.absolute()
+    output_dir.mkdir(exist_ok=True)
```

Comment:
I would add the argument `parents=True` to make sure all parents directories are created.